spec/factories/gardens.rb

```diff
@@ -1,7 +1,6 @@
 FactoryBot.define do
   factory :garden do
-    zip_code { "MyString" }
-    name { "MyString" }
-    user { nil }
+    sequence(:zip_code) { |n| "8020#{n.to_s}" }
+    sequence(:name) { |n| "Front Yard#{n}" }
```

Comment:
Looks like you accidentally deleted the user from here. I think if you resolve the merge conflicts and use the settings in master, it should work, or just add in `user`.

Reply:
Nice catch! I resolved merge conflict and insured user was present on the garden FactoryBot.